init.go

```diff
@@ -10,11 +10,12 @@ import (
 	"github.com/opencontainers/runc/libcontainer/logs"
 	_ "github.com/opencontainers/runc/libcontainer/nsenter"
 	"github.com/sirupsen/logrus"
-	"github.com/urfave/cli"
 )
 
 func init() {
 	if len(os.Args) > 1 && os.Args[1] == "init" {
+		// This is the golang entry point for runc init, executed
+		// before main() but after libcontainer/nsenter's nsexec().
```

Comment:
Might not hurt to mention the function never returns (since this all ends in `execve`) so `main` never actually runs.

Reply:
I had something like that but eventually removed it as this is somewhat clear (for one thing, the code within the block ends with a `panic`).

I will try to improve this further in #3114 (which is a followup to this PR).